diff --git a/mock-core-configs/etc/mock/templates/centos-8.tpl b/mock-core-configs/etc/mock/templates/centos-8.tpl
index 44389809..e0861652 100644
--- a/mock-core-configs/etc/mock/templates/centos-8.tpl
+++ b/mock-core-configs/etc/mock/templates/centos-8.tpl
@@ -46,6 +46,14 @@ mirrorlist=http://mirrorlist.centos.org/?release=8&arch=$basearch&repo=PowerTool
 gpgcheck=1
 gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Official
 
+[Devel]
+name=CentOS-$releasever - Devel WARNING! FOR BUILDROOT USE ONLY!
+mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=Devel&infra=$infra
+#baseurl=http://mirror.centos.org/$contentdir/$releasever/Devel/$basearch/os/
+gpgcheck=1
+enabled=0
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Official
+
 [centosplus]
 name=CentOS-$releasever - Plus
 mirrorlist=http://mirrorlist.centos.org/?release=8&arch=$basearch&repo=centosplus&infra=$infra
